src/main/scala/org/graphframes/lib/ShortestPaths.scala

```diff
@@ -17,13 +17,14 @@
 
 package org.graphframes.lib
 
-import scala.reflect._
-import scala.reflect.runtime.universe._
+import java.util
 
-import org.apache.spark.graphx.{lib => graphxlib}
+import scala.collection.JavaConversions._
```

Comment:
I learned something about scala, thanks! JavaConverters makes the conversion explicit.